sbndaq-artdaq/Generators/Common/CAENV1730Readout_generator.cc

```diff
@@ -1014,6 +1063,9 @@ void sbndaq::CAENV1730Readout::ConfigureTrigger()
     }
   }
 
+  // for SBND
+  if( (fOutputClk == 1) | (fOutputClkPhase == 1) ){ ConfigureClkToTrgOut(); } 
+
```

Comment:
The `ConfigureClkToTrgOut()` takes good care of avoiding to mess up the [0:15] bits of the `FP_IO_CONTROL` register, however the following function `ConfigureLVDS()` (relevant only for ICARUS) doesn't seem to do the same... It builds this `ioMode` and then sets it in the register without caring what was there before.

I believe it would not be possible for ICARUS to use this new feature if we keep the current ordering because the next function overrides the register. Why not moving your call after the call to `ConfigureLVDS()`?

Also a minor detail, since these are booleans:
```suggestion
  // for clock synchronization studies
  if( fOutputClk || fOutputClkPhase ){ ConfigureClkToTrgOut(); } 

```